src/pages/overview/pdf-services-api/howtos/protect-pdf.md

```diff
@@ -34,36 +34,40 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
         public static void main(String[] args) {
      
             try {
-                // Initial setup, create credentials instance.
-                Credentials credentials = Credentials.servicePrincipalCredentialsBuilder()
-                    .withClientId("PDF_SERVICES_CLIENT_ID")
-                    .withClientSecret("PDF_SERVICES_CLIENT_SECRET")
-                    .build();
-     
-                // Create an ExecutionContext using credentials.
-                ExecutionContext executionContext = ExecutionContext.create(credentials);
-     
-                // Build ProtectPDF options by setting a User Password and Encryption
-                // Algorithm (used for encrypting the PDF file).
-                ProtectPDFOptions protectPDFOptions = ProtectPDFOptions.passwordProtectOptionsBuilder()
-                        .setUserPassword("encryptPassword")
+                // Initial setup, create credentials instance
+                Credentials credentials = new ServicePrincipalCredentials(
+                        System.getenv("PDF_SERVICES_CLIENT_ID"),
+                        System.getenv("PDF_SERVICES_CLIENT_SECRET"));
+    
+                // Creates a PDF Services instance
+                PDFServices pdfServices = new PDFServices(credentials);
+    
+                // Creates an asset from source file and upload
+                InputStream inputStream = Files.newInputStream(new File("src/main/resources/protectPDFInput.pdf").toPath());
+                Asset asset = pdfServices.upload(inputStream, PDFServicesMediaType.PDF.getMediaType());
+    
+                // Create parameters for the job
```

Comment:
same

Reply:
As discussed, IG this is from already reviewed samples.